hummingbot/client/config/config_validators.py

```diff
@@ -1,16 +1,16 @@
-from hummingbot.core.utils.trading_pair_fetcher import TradingPairFetcher
 from decimal import Decimal
 from typing import Optional
-from hummingbot.client.settings import EXCHANGES, STRATEGIES
```

Comment:
Changing these imports to be within each function seems to be well worth doing though to fix circular references when importing `validate_bool` in connectors e.g. `bamboo_relay`

Reply:
I think this is correct. @vic-en Can you please review this?

Reply:
yes, that's correct.
It'd prevent one of the circular errors.